test/gtest/ucs/ucx_test.conf

```diff
@@ -1,3 +1,26 @@
 UCX_PRICE=200
 UCX_BRAND=Mazda
+
+[section to use]
+CPU vendor=*
+CPU model=*
+Product name=*
 UCX_TEMP=10,front:15
+
+[section to skip 1]
+CPU vendor=Skip section
+CPU model=*
+Product name=*
+UCX_PRICE=300
```

Comment:
can you pls provide some example how this skip feature will be used?

Reply:
Section is used, if filtering patterns are matched.
E.g. we have the following config
```
[section1]
CPU model=*model_1*
UCX_VALUE=value_1

[section2]
Product name=*product_2*
UCX_VALUE=value_2
```
, then:
```
$ ucx_info -s
...
# CPU vendor: vendor_1
# CPU model: model_1 update
# Product name: product_1
...
$ ucx_info -c | grep UCX_VALUE
UCX_VALUE=value_1
```
, or:
```
$ ucx_info -s
...
# CPU vendor: vendor_2
# CPU model: model_2
# Product name: product_2
...
$ ucx_info -c | grep UCX_VALUE
UCX_VALUE=value_2
```